boards/arm/nucleo_g431rb/doc/index.rst

```diff
@@ -170,9 +170,10 @@ is selected.
 Serial Port
 -----------
 
-Nucleo G431RB board has 3 U(S)ARTs. The Zephyr console output is assigned to UART2.
+Nucleo G431RB board has 3 U(S)ARTs and one LPUART. The Zephyr console output is assigned to LPUART1.
 Default settings are 115200 8N1.
 
+Please note that LPUART1 baudrate is limited to 9600 if it is clocked by LSE.
```

Comment:
Replace "it" with whatever it refers to, to make language more clear.

Reply:
Done.